### PR TITLE
Split terraform state into multiple layers

### PR DESCRIPTION
**Pull request by nargiza777**

### CODE REVIEW

File: level2/terraform.tfvars [hunk 1]
```diff
@@ -0,0 +1 @@
+env_code = "orlando"
```
Comment (geoffrey1330) [inline level2/terraform.tfvars:1]: could you please, add a newline at the end of file.

File: level2/variables.tf [hunk 2]
```diff
@@ -0,0 +1 @@
+variable "env_code" {}
```
Comment (geoffrey1330) [inline level2/variables.tf:1]: could you please, add a newline at the end of file.

**Review by geoffrey1330 — CHANGES_REQUESTED** [review]

File: level2/terraform.tfvars (hunk 1 above)
Comment (nargiza777) [reply level2/terraform.tfvars]: added

File: level2/variables.tf (hunk 2 above)
Comment (nargiza777) [reply level2/variables.tf]: added

**Review by geoffrey1330 — APPROVED** [review]

**Pull request merged**